vmdb/app/models/ems_refresh/parsers/openstack_infra.rb

```diff
@@ -56,25 +72,35 @@ def load_hosts
         indexed_hosts_ports = {}
         hosts_ports.each { |p|  (indexed_hosts_ports[p.uuid] ||= []) <<  p }
 
+        # Indexed Heat resources, we are interested only in OS::Nova::Server
+        indexed_resources = {}
+        resources.each { |p| indexed_resources[p['physical_resource_id']] = p }
+
         process_collection(hosts, :hosts) do  |host|
-          parse_host(host, indexed_servers, indexed_hosts_ports)
+          parse_host(host, indexed_servers, indexed_hosts_ports, indexed_resources)
         end
       end
 
-      def parse_host(host, indexed_servers, _indexed_hosts_ports)
+      def parse_host(host, indexed_servers, _indexed_hosts_ports, indexed_resources)
         uid = host.uuid
+        hostname = identify_hostname(indexed_resources, host.instance_uuid)
```

Comment:
I would like to have it either in name or hostname. Maybe name is better fit? Nevertheless only these 2 params shows as hover in host list. It's nice so see this information there.